src/main/java/com/microsoft/azure/functions/worker/Application.java

```diff
@@ -122,7 +123,7 @@ public static void main(String[] args) {
             try (JavaWorkerClient client = new JavaWorkerClient(app)) {
                 client.listen(app.getWorkerId(), app.getRequestId()).get();
             } catch (Exception ex) {
-                WorkerLogManager.getSystemLogger().log(Level.SEVERE, "Unexpected Exception causes system to exit", ex);
+                WorkerLogManager.getSystemLogger().log(Level.SEVERE, ExceptionUtils.getRootCauseMessage(ex), ex);
```

Comment:
It this an unrelated change?

Reply:
Yes. Improving error logging to help us get the exact exception message